themes/vocabulary_theme/templates/blocks/featured-projects.html

```diff
@@ -0,0 +1,46 @@
+<section class="featured-projects">
+  <div class="container">
+    <div class="level">
+      <h4 class="is-paddingless level-left">
+        {{ this.title }}
+      </h4>
+      <span class="level-right">
+        <a class="project-page-link" href="/contributing-code/projects/">See all available projects <i class="icon angle-right"></i></a>
+      </span>
+    </div>
+    <div class="project-list columns">
+      {% for repo in bag('repos.repos') %}
+      {% if repo.featured %}
+        <article class="card entry-post vertical project-index column is-one-third">
```

Comment:
Sames as blog post... let's try to keep separate the column container from the component

```
<div class="columns">
  <div class="column is-one-third">
    <article class="card entry-post vertical project-index">
      [...]
    </article>
  </div>
```

Reply:
Alright! And about the same height thing, I followed the designs actually, you can see it [here](https://www.figma.com/file/mttcugI1UxvCJRKE5sdpbO/Mockups?node-id=759%3A516). And if you want me to change them to same height, I just have to remove `height: 100%`

Reply:
And regarding the third featured project - I think I should make changes in the master (through a PR). So I will raise a new PR for that. Does that make sense to you?